Minor bug fixes and enhanced scikit-bio compliance

dparks1134 opened the pull request:
- fixes bug where first line of consensus map is improperly read
- allows taxa to be separated by a comma in addition to a comma+space in the consensus map 
- better compliance with scikit-bio, especially within the unit tests

wasade commented on scripts/t2t:
```diff
@@ -138,30 +142,30 @@ def validate(taxonomy, limit, flat_errors, hierarchy_errors):
-    
+
     if not quiet:
         print 'Determining taxonomic consistency of: '
```
Non-blocking, but in hindsight, these print statements should be `click.echo` instead. Primary reason is that, if these interfaces end up using click's testrunner, then it can readily suppress the output.

wasade commented on scripts/t2t:
```diff
@@ -138,30 +142,30 @@ def validate(taxonomy, limit, flat_errors, hierarchy_errors):
 @click.option('--quiet', is_flag=True, default=False, help='Suppress output')
```
should be `--verbose`

wasade commented on t2t/nlevel.py:
```diff
@@ -553,6 +560,7 @@ def set_preliminary_name_and_rank(tree):
 
 def make_consensus_tree(cons_split, check_for_rank=True, tips=None):
```
This method is actually redundant at this time as `TreeNode` has a `from_taxonomy` method, but I don't know off hand how easy it would be to drop in here. Just making a general comment, not blocking

wasade commented:
Seems good, thanks! Looks like there are a few pep8 issues that are triggering the failure, and just a few minor comments

dparks1134 commented:
Moved to click.echo() so we can start following that as a pattern. Changed parameter to verbose instead of quiet. Updated validation.py for PEP-8 compliance.

Also now realizing why it is bad, very bad to develop on the master branch. Live and learn!

sjanssen2 commented:
@wasade @dparks1134 I added some fixed to this PR and launch a new one: PR #28
